5.14.5-openshift-v3.9.14/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM openjdk:8-jre
+
+ENV ACTIVEMQ_VERSION 5.14.5
+ENV ACTIVEMQ apache-activemq-$ACTIVEMQ_VERSION
+ENV ACTIVEMQ_TCP=61616 ACTIVEMQ_AMQP=5672 ACTIVEMQ_STOMP=61613 ACTIVEMQ_MQTT=1883 ACTIVEMQ_WS=61614 ACTIVEMQ_UI=8161
+
+ENV ACTIVEMQ_HOME /app/apache-activemq-5.14.5
+
+WORKDIR /app
+
+RUN set -x && \
+    curl -s -S https://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /app 
+
+RUN chgrp -R 0 /app && \
```

Comment:
hi
Will do. sorry for the late reply.

Reply:
hi Again...I think if you can first please merge the PR for version 5.14.5 that would be good. That way at least I will know it will not be another year before I get a reply ;=)
I will then update the latest image with an OC compliant one.

Reply:
It took me a few months to realize that I missed an issue. But to be fair it took also 7 months until you responded again. :)

I don't see much value in having extra openshift images, if only that line is missing. Just add it directly to existing images and I can push them.

Feel free to ping me if I don't react in a reasonable time-frame.

Reply:
I think we should retorfit say 3 versions backwardly with an OC branch.
That way, projects across the world, when moving their INFRa to OC can use the image.

Reply:
works for me.